node/validation/manifest/index.test.ts

```diff
@@ -51,6 +51,21 @@ test('should throw ManifestValidationError with correct message', () => {
   expect(() => validateManifest('manifest')).toThrowError(/^Validation of Edge Functions manifest failed/)
 })
 
+test('should throw ManifestValidationError with customErrorInfo', () => {
+  try {
+    validateManifest('manifest')
+  } catch (error) {
+    expect(error).toBeInstanceOf(ManifestValidationError)
+
+    const { customErrorInfo } = error as ManifestValidationError
+    expect(customErrorInfo).toBeDefined()
```

Comment:
nit: This is redundant given the next assertion.

Reply:
Will fix in the next PR